website/content/getting-started/deployment.mdx

```diff
@@ -0,0 +1,277 @@
+---
+id: getting-started-deployment
+title: Deployment - Getting started
+permalink: /getting-started/deployment
+---
+
+import { ContentNotification } from '@commercetools-frontend/ui-kit';
+
+<ContentNotification type="info">
+  This feature is marked as `beta` and may be affected by changes. Use with caution for production.
+</ContentNotification>
+
+# Deploying a Merchant Center application
+
+After you've begun developing your application, it's time to think about deployment for testing and production purposes. This section introduces what you need to do to prepare your application for production deployment, and provides deployment examples.
+
+## Prepare the application for production use
+
+In order to run the application in `production` mode, we need to **build** it using the following command:
+
+```
+$ yarn build
+```
+
+![Production build](/assets/custom-applications/mc-dev-yarn-build.png)
```

Comment:
Images now can be rendered like this. Under the hood, gatsby will transform this into an optimized image. Images used for content pages should be defined in `./static/assets` folder.